PyPlay.js

```diff
@@ -3,6 +3,7 @@ document.addEventListener("DOMContentLoaded", () => {
     ace.require("ace/ext/language_tools");
     const editor = ace.edit("editor");
     editor.session.setMode("ace/mode/python");
+	editor.setShowPrintMargin(false);
```

Comment:
Looks like the indentation is off, can you set up your editor to use Prettier?

Reply:
Ok I switched editors to atom and installed Prettier.  It seems to modify the file correctly.  Going to see if I can push the changes now.

Reply:
Sorry I am not sure if I "pushed" the changes correctly.  Please let me know if I did not.

Reply:
I don't see any additional commits.